### ISSUE TITLE
Shortcuts do nothing for me anymore.

### ISSUE THREAD

**Issue by ColetteDiskette** [Merrit/kwin-move-window#4]
What the title says. If I press Meta+Num+2 for instance, nothing happens. The script is enabled and the shortcuts are set. This worked fine for me until somewhat recently. I've tried disabling and re-enabling the script as well as re-setting the keybinds. (Running Debian testing, Plasma 5.27.10, Kernel 6.6.15-amd64. Dual monitor setup with one vertical monitor.)

**Comment by Merrit** [comment]
What version of  the script is installed? If you are still on Plasma 5 you should be using  [v1.0.2](https://github.com/Merrit/kwin-move-window/releases/tag/v1.0.2).

v1.1.0 and up are only compatible with Plasma 6. I would hope the store wouldn't upgrade someone to an unsupported version..

**Comment by ColetteDiskette** [comment]
It seems it certainly did. I was on v1.1.1. I just uninstalled it to try to reinstall and now the script seems to just not be in the store for me whatsoever. I downloaded the version you linked and now it says I have v1.0.0 installed but still doesn't seem to work for me for whatever reason. 🙁

**Comment by Merrit** [comment]
That's a pretty big oversight for the Pling store, if they filter out new installs for supported version but not updates.. :roll_eyes: 

It'll say v1.0.0 for the version  I linked because it wasn't set in that metadata previously.

Because of the whole update/downgrade thing it might be a bit confused. If you haven't, try rebooting and disable/enable the script. If it still isn't working, try manually re-binding one of the shortcuts to see if that helps..

If still no luck, we can try to check debug output.

**Comment by ColetteDiskette** [comment]
It works again! What I did was: disable the script, reboot, enable the script. I didn't try anything lesser though, so it's possible just disable/enable would've worked and I was too out of it last night to even think of something that simple. Either way, it works again! Much appreciated!

Still troublesome that I managed to be fed the update when I shouldn't have been, but hopefully this issue being here will help anyone else who had the same issue. 😄
